### PR TITLE
@duckdb/duckdb-wasm 1.24

### PR DESCRIPTION
**Pull request by mbostock**
Supersedes #350.

### CODE REVIEW

File: src/duckdb.js [hunk 1]
```diff
@@ -169,7 +171,7 @@ async function insertFile(database, name, file, options) {
     const buffer = await file.arrayBuffer();
     await database.registerFileBuffer(file.name, new Uint8Array(buffer));
   } else {
-    await database.registerFileURL(file.name, url, duckdb.DuckDBDataProtocol.HTTP);
+    await database.registerFileURL(file.name, url, 4); // duckdb.DuckDBDataProtocol.HTTP
```
Comment (domoritz) [inline src/duckdb.js:174]: This could be an issue if the enum ever changes.

Comment (mbostock) [reply src/duckdb.js:174]: Yes. Something to check when we upgrade.

Comment (domoritz) [reply src/duckdb.js:174]: Is there a reason not to use the value from the DuckDB lib?

Comment (mbostock) [reply src/duckdb.js:174]: Just plumbing. The `duckdb` module isn’t available globally; it’s loaded in `loadDuckDB` here:

https://github.com/observablehq/stdlib/blob/89f27007453ea1919493dca3fb94b7a5bde03222/src/duckdb.js#L256

So, I’d need to call `loadDuckDB` again here, or I’d need to somehow pass it through the call to `createDuckDB` above and then pass it down into `insertFile`.

https://github.com/observablehq/stdlib/blob/89f27007453ea1919493dca3fb94b7a5bde03222/src/duckdb.js#L127

https://github.com/observablehq/stdlib/blob/89f27007453ea1919493dca3fb94b7a5bde03222/src/duckdb.js#L138

All of those felt like a lot more work than typing the number `4`.

**Pull request merged**